lib/Echidna/Solidity.hs

```diff
@@ -96,16 +97,17 @@ contracts fp = let usual = ["--solc-disable-warnings", "--export-format", "solc"
   a  <- view (hasLens . solcArgs)
   q  <- view (hasLens . quiet)
   ls <- view (hasLens . solcLibs)
+  c  <- view (hasLens . cryticArgs)
   let solargs = a ++ linkLibraries ls & (usual ++) . 
                   (\sa -> if null sa then [] else ["--solc-args", sa])
   maybe (throwM CompileFailure) (pure . toList . fst) =<< liftIO (do
     stderr <- if q then UseHandle <$> openFile "/dev/null" WriteMode else pure Inherit
-    _ <- readCreateProcess (proc "crytic-compile" $ solargs |> fp) {std_err = stderr} ""
+    _ <- readCreateProcess (proc "crytic-compile" $ (c : solargs) |> fp) {std_err = stderr} ""
```

Comment:
in particular we want `c = cryticArgs` to be a list of arguments here, instead of a massive string containing multiple arguments, otherwise we run `$ crytic-compile "$cryticArgs" --solc-args "$solcArgs" "$fp"` which is incorrect.